packages/react-components/react-teaching-popover-preview/src/components/TeachingPopover/TeachingPopover.types.ts

```diff
@@ -0,0 +1,43 @@
+import type { PopoverState, PopoverProps } from '@fluentui/react-popover';
+
+/**
+ * TeachingPopover Props
+ */
+export type TeachingPopoverProps = Omit<PopoverProps, 'appearance'> & {
+  /**
+   * Enables user to dictate current page number via props
+   */
+  currentPage?: number;
+  /**
+   * Callback to notify a page change (can be used to update 'currentPage' externally).
+   */
+  onPageChange?: (index: number) => void;
```

Comment:
These callbacks are not compliant with Fluent: they should have event & `data` as arguments.